### DISCUSSION TITLE
Lazer HUD or UI/maybe graphics

### DISCUSSION THREAD

**Discussion by Fraiits** [ppy/osu#21671] [Q&A]
So sorry I don't know much about fixing/making games yet, but while trying out Osu!Lazer again after hearing about new performance updates I wanted to go back and test how it felt. 
I have a Rtx 2070 Super, Ryzen 9 3900x, 34gb 3600mhz ram, BenQ 240hz monitor and other stuffs. 
After the new update it didn't feel like anything was off until I played a map or two. At first it felt like my aim was off a bit almost like it was stretched different or the cursor felt behind/laggy. So I changed a bunch of settings, learned them and put performance to the max, but I noticed the "laggy" feeling didn't go away. So assuming that's how it was I just kept playing until I added more Ui to the HUD. (Note I was also streaming and noticed that the problem was the same regardless of me streaming or not, so not hardware bound I think)

I noticed the problem increased and the game was actually lagging then. So to really make sure I opened up Osu! with the best performance my pc/windows can supply and it felt so much smoother, and I noticed it instantly going into a song.

So an idea came to mind, and that was just to hid the HUD. As soon as I did that the game felt almost exactly as smooth as Osu! if not way better. I'm not sure why that would be the case (maybe too much of something) but I could tell it was fixed and you can even hear my excitement in my stream. I also had a friend trying out Osu!Lazer in stream and he also said that hiding the HUD made the game feel much better. Although what confused me about the problem is when he stated that turning it back on didn't bring the problem back until you start adding Ui or editing your skin in game.

I need to share all my settings + Nvidea settings I would be more than glad to as I had a lot of fun once it felt much better. Very excited for this to be the main game because it has a lot of things that I like, a small one would be how the sound quiets down when you TAB out xP.

This is where I started the initial discussion. and that screenshot just shows my fps, and that was the lowest I have ever seen it go to in Lazer. After chaning the HUD I never saw it go below 890-870
[https://twitter.com/Fraiits/status/1603227404071313409?s=20&t=qm89CD9eYYPVPao3QOGatw](url)
![Screenshot 2022-12-16 132654](https://user-images.githubusercontent.com/120746035/208164682-db962cd5-2dfb-46a6-9309-37bc7ae30990.png)

@peppy

**Comment by Fraiits** [comment]
If I were a new person trying it out rn and didn't know the steps to fix it, it might have detoured me away. I assume the game will feel much better when it's closer to it's final form, but I was just hoping this could maybe help a tad or if not then I tried at least. Also I wasn't sure what to Categorize the discussion, so I put it in Q&A, lmk if I need to change it.

**Comment by peppy** [comment]
Please provide more details:

- Which skin are you using? If you have customized the HUD please export and attach the skin.
- What ruleset are you playing on?
- Hit ctrl-shift-s and remove HUD elements one by one. You may be able to figure which component in the HUD is causing you issues.
